guides/v2.0/extension-dev-guide/extension_attributes/adding-attributes.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: default
+group: extension-dev-guide
+subgroup: 99_Module Development
+title: Adding extension attributes to entity
+menu_title: Adding extension attributes to entity
+menu_order: 20
+version: 2.0
+github_link: extension-dev-guide/extension_attributes/adding-attributes.md
+redirect_from: /guides/v2.0/extension-dev-guide/extension_attributes/adding-attributes.html
+---
+
+##{{page.menu_title}}
+
+Sometimes we faced with situations when we need to add additional data to entities like product.
+Off course, we can use custom attributes for this purposes, however this can be impossible in some situations, for instance, if you have this attributes stored in Mongo Db. 
```

Comment:
The main reason why it might not be possible for third-party developers is that they cannot change directly API Data interfaces in the Magento core. That's why extension attributes is the only way to affect interfaces using configuration. I would remove statement about Mongo because most developers will think "ah, I don't use Mongo with Magento, that's not for me"